Meadowlark-js/tests/system/crud/Delete.test.ts

```diff
@@ -3,37 +3,46 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
-import { deleteIt, FrontendResponse, get, upsert } from '@edfi/meadowlark-core';
-import { getNewClient, getCollection, resetSharedClient } from '@edfi/meadowlark-mongodb-backend';
-import { MongoClient } from 'mongodb';
+import { deleteIt, FrontendResponse, upsert, get } from '@edfi/meadowlark-core';
 import {
   schoolBodyClient1,
-  schoolDeleteClient2,
   schoolGetClient1,
   schoolDeleteClient1,
-  academicWeekBodyClient1,
   configureEnvironmentForMongoSystemTests,
+  academicWeekBodyClient1,
+  schoolDeleteClient2,
 } from './SystemTestSetup';
 
+let backendToTest;
+
+(async () => {
+  // TODO - Remove the following lines - this is for being able to test within VSCode due to no environment set
+  process.env.DOCUMENT_STORE_PLUGIN = '@edfi/meadowlark-postgresql-backend';
+  // process.env.DOCUMENT_STORE_PLUGIN = '@edfi/meadowlark-mongodb-backend';
+  const plugin = process.env.DOCUMENT_STORE_PLUGIN ?? '@edfi/meadowlark-mongodb-backend';
+  backendToTest = await import(plugin);
+})();
+
 jest.setTimeout(40000);
 configureEnvironmentForMongoSystemTests();
 
 describe('given a DELETE of a non-existent school', () => {
-  let client: MongoClient;
+  let client;
   let deleteResult: FrontendResponse;
 
   beforeAll(async () => {
-    client = (await getNewClient()) as MongoClient;
-    await getCollection(client).deleteMany({});
+    client = await backendToTest.TestingSetup();
```

Comment:
This is the other oddity I mentioned Monday, With Mongo, if I try to end the pool in the setup function and then recreate in the teardown it causes connection issues, if I leave the client w/o passing it to the teardown it hangs Jest. The only way I could get it to work is to pass the client to the teardown and close it there. This code technically works and meets the tickets requirements for not having backend specific code in the tests, but it is a little weird.